server/docs/hellobooks_api_doc.yaml

```diff
@@ -945,12 +943,6 @@ paths:
         - bearerAuth: []
       summary: Get user profile
       description: User can get other user profiles including theirs
-      parameters:
-        - in: query
-          name: id
-          schema:
-            type: integer
-          required: true 
```

Comment:
I'm guessing this param was to get other user's profile too.
Therefore this should be here but not required

Reply:
@encodedBicoding

Reply:
That is true, it should be false. @cvjude give the required property value of false

Reply:
The userId was implemented in the request query.

Reply:
What do you mean @cvjude

Reply:
So I am guessing the route serves two purposes, if the ID is provided, it gets the user with given ID profile else it gets the logged in user profile

Reply:
Yes, that is what it does